js/src/reports/metric-number.js

```diff
@@ -48,19 +49,43 @@ const MetricNumber = ( {
 		const formatFn = isCurrency ? formatAmount : formatNumber;
 
 		return {
-			value: formatFn( value ),
+			value:
+				value === undefined
+					? __( 'Unavailable', 'google-listings-and-ads' )
+					: formatFn( value ),
 			prevValue: formatFn( prevValue ),
 		};
 	}, [ isCurrency, value, prevValue, formatNumber, formatAmount ] );
 
 	let markedLabel = label;
+	let infoText;
 
 	// Until ~Q4 2021, metrics for all programs, may lack data for free listings.
-	if ( missingFreeListingsData ) {
-		const infoText = __(
-			'This data is available for paid campaigns only.',
+	// And Free Listings API may not respond with data.
+	if ( missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_METRIC ) {
+		infoText = __(
+			'This data is currently available for paid campaigns only.',
 			'google-listings-and-ads'
 		);
+	} else if (
+		missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_REQUEST
+	) {
+		infoText = (
+			<>
```

Comment:
:construction: 
This breaks `aria-label` used below, as it renders `"aria-label="[object Object]"`